pex/variables.py

```diff
@@ -339,28 +339,6 @@ def PEX_VERBOSE(self):
     """
     return self._get_int('PEX_VERBOSE', default=0)
 
-  # TODO(#94) Remove and push into --flags.
-  @property
-  def PEX_HTTP_RETRIES(self):
-    """Integer
-
-    The number of HTTP retries when performing dependency resolution when building a PEX file.
-    Default: 5.
-    """
-    return self._get_int('PEX_HTTP_RETRIES', default=5)
-
-  # TODO(#94) Remove and push into --flags.
-  @property
-  def PEX_HTTP_TIMEOUT(self):
-    """Integer
-
-    When built with the `requests` HTTP library, a timeout to apply for HTTP connect attempts and
-    read attempts. See the `requests` docs (https://2.python-requests.org/en/master/user/quickstart/#timeouts)
-    for more information.
-    Default: 15.
-    """
-    return self._get_int('PEX_HTTP_TIMEOUT', default=15)
```

Comment:
Might want to make this a dedicated PR so that Pex's changelog can say "Remove unused `PEX_HTTP_RETRIES` and `PEX_HTTP_TIMEOUT` flags." Otherwise, this is easy to miss.

Reply:
I'd like to keep this inline since nothing breaks - these are just silent env var non-reads that have been happening since October. I will call it out in the release header though since they're both gone and were always a bad idea for the reasons Wickman notes in #94.